CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
Is it safe to add CMakeLists.txt to top directory of repository that is going to be a part of NCS? I'm not sure if we can expect any potential conflicts in build system IDEs, etc.

Reply:
I don't see any reason for it to cause any problems. Zephyr modules do not magically pull any CMakeLists file unless explicitly told to do so. This file can be pulled in a controlled way in an equivalent fashion as with OpenThread
https://github.com/zephyrproject-rtos/openthread/blob/1d668284a0b897bd6d539e7c3ed8009f0028099d/zephyr/CMakeLists.txt#L229

Reply:
Most probably my doubt is caused by the CMakeLists file used in Zephyr that was present in hal_nordic repository in top 802.15.4 directory. But now as this file is going to be moved to Zephyr upstream tree, I don't see conflicts anymore. Thanks for explaining.